pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md

```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
 After you have the available Konvoy versions, you can upgrade your CLI by running the following command:
 
 ```bash
-konvoy image upgrade --version=v1.8.4
-Wrote Konvoy CLI version 'v1.8.4' to '.konvoy/cli_version'
+konvoy image upgrade --version=v1.8.5
+Wrote Konvoy CLI version 'v1.8.5' to '.konvoy/cli_version'
 ```
 
 After the upgrade command completes, you can start using the new Konvoy version.
 
-### Upgrading Konvoy from v1.7.x to v1.8.4
+### Upgrading Konvoy from v1.7.x to v1.8.5
 
 **You must modify your `cluster.yaml` with these changes when upgrading from a previous Konvoy version:**
 
 It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for of kind: ClusterConfiguration.
 
-The version of Calico got bumped in Konvoy v1.8.4, from Calico v3.17.3 to v3.20.2. You will want to change this in `spec.containerNetworking.calico.version: v3.17.3` for of kind: ClusterConfiguration.
+The version of Calico got bumped in Konvoy v1.8.5, from Calico v3.17.3 to v3.20.2. You will want to change this in `spec.containerNetworking.calico.version: v3.17.3` for of kind: ClusterConfiguration.
```

Comment:
I believe this is incorrect, the Calico bump happened in 1.8.4, not 1.8.5.  

```suggestion
The version of Calico was bumped in Konvoy v1.8.4, from Calico v3.17.3 to v3.20.2.  When upgrading, set `spec.containerNetworking.calico.version: v3.20.2` for kind: ClusterConfiguration.
```

Reply:
Good catch with that wording @cneth